packages/config/config/devices/0x0460/templates/wave_template.json

```
@@ -360,6 +360,74 @@
 		"maxValue": 32535,
 		"defaultValue": 30
 	},
+		"operating_modes": {
+		"label": "Operating Modes",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Shutter",
+				"value": 0
+			},
+			{
+				"label": "Venetian blind",
+				"value": 1
+			},
+			{
+				"label": "Set Manual time",
+				"value": 2
+			}
+		]
+	},
+		"motor_limit_detection": {
+		"label": "Motor Limit Detection",
+		"description": "Power threshold used to detect a limit switch. Increase this value if the device falsely detects limit switches.",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"defaultValue": 1,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Auto calibration",
+				"value": 1
+			}
+		]
+	},
+		"delay_limit_detection": {
+		"label": "Limit Detection Delay Time",
+		"description": "Defines the delay before motor power is read after a relay switches ON",
+		"unit": "0,1 sec",
+		"valueSize": 1,
+		"minValue": 3,
+		"maxValue": 50,
+		"defaultValue": 3,
+		"options": [
+			{
+				"label": "Automatic time",
+				"value": 0
+			}
+		]
+	},
+		"max_moving_time": {
+		"label": "Shutter Max Moving Time",
+		"description": "Defines the maximum time the motor can move before stopping",
+		"unit": "seconds",
+		"valueSize": 2,
+		"minValue": 1,
+		"maxValue": 32000,
+		"defaultValue": 120,
```

This parameter may also need

```suggestion
		"defaultValue": 120,
		"unsigned": true,
```
if the checks complain about incompatible min/max value after the change.